app/helpers/application_helper.rb

```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
+  def bootstrap_class_for_toast(type)
+    case type
+    when 'error'
+      'text-bg-danger'
+    when 'success'
```

Comment:
sikeres belépéskor a devise `notice` típust használ. lehet ezt kellene nekünk is, hogy midenhol jól nézzen ki?